models.py

```diff
@@ -201,11 +201,49 @@ def store(cls, institution, session_name):
   def delete(cls, institution, session_name):
     Session.session_key(institution, session_name).delete()
 
+class ServingRules(ndb.Model):
+  """List of serving rules in yaml and json format."""
+  data = ndb.TextProperty()
+  jdata = ndb.JsonProperty()
+
+  @classmethod
+  @timed
+  def serving_rules_key(cls, institution, session):
+    return ndb.Key("InstitutionKey", institution,
+                   Session, session,
+                   ServingRules, "serving_rules")
+
+  @classmethod
+  @timed
+  def FetchJson(cls, institution, session):
+    serving_rules = ServingRules.serving_rules_key(institution, session).get()
+    if serving_rules:
+      return serving_rules.jdata
+    else:
+      return ''
+
+  @classmethod
+  @timed
+  def Fetch(cls, institution, session):
+    serving_rules = ServingRules.serving_rules_key(institution, session).get()
+    if serving_rules:
+      return serving_rules.data
+    else:
+      return ''
+
+  @classmethod
+  @timed
+  def store(cls, institution, session_name, sr_data):
+    serving_rules = ServingRules(data = sr_data,
+                                 jdata = yaml.load(sr_data))
+    serving_rules.key = ServingRules.serving_rules_key(institution, session_name)
+    serving_rules.put()
+
 
 class ServingSession(ndb.Model):
   """Which session is currently serving. Empty if none."""
   session_name = ndb.StringProperty()
-  login_type = ndb.StringProperty(choices=['verification', 'preferences', 'schedule', 'preregistration', 'postregistration'])
+  start_page = ndb.StringProperty(choices=['verification', 'preferences', 'schedule', 'preregistration', 'postregistration'])
```

Comment:
I'm shocked that this works. Does it?  After you made the change was the old data recognized? I imagine we would have to write a db upgrade script after this change.